packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts

```diff
@@ -33,8 +33,15 @@ export class TypeScriptFileUpdate {
 		this.initState();
 	}
 
-	public addRoute(filePath: string, linkPath: string, linkText: string, routesVariable = DEFAULT_ROUTES_VARIABLE) {
-		this.addRouteModuleEntry(filePath, linkPath, linkText, routesVariable);
+	public addRoute(
```

Comment:
This method is actually doing two things, at least. It adds routes as well as it adds imports. We should try to split this into two separate methods. Note, to do this we may need to create separate visitor for imports and routes, and all this should be done in `core` `TypeScriptFileUpdate`.

Reply:
I added this to the requirements in issue https://github.com/IgniteUI/igniteui-cli/issues/983